### PR TITLE
Replace `File.exists?` with `File.exist?`

### PR DESCRIPTION
**Pull request by tom-lord**
Fixes a deprecation warning.

`File.exists?` will be removed in ruby 3.2
(Although I don't really agree with that decision!!...)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -112,3 +112,6 @@ DEPENDENCIES
   rails (= 3.0.9)
   rb-fsevent (>= 0.9.2)
   rspec-rails (~> 2.11)
+
+BUNDLED WITH
+   1.17.3
```
Comment (tom-lord) [inline Gemfile.lock:117]: This (old) gem specifies a development dependence of Rails 3.x, which isn't compatible with bundle 2.x

I didn't want to take on the scope creep of fixing that too...

**Review by shamess — APPROVED** [review]
